### ISSUE TITLE
RDAdmin: Missing Icon on Groups List Add and some crashing

### ISSUE THREAD

**Issue by alexolivan** [ElvishArtisan/rivendell#701]
HI!

Ive finally setup my nightly building system of qt5-branch snapshot debs in a private repo... so I can help in testing :-)
I'm setting up a new radio from scratch on a test VM, to test everything!... now for the first issues: 

- When creating groups I've noticed that the (very nice!) side icon on the list is not rendered upon Group creation... so it looks weird newly created groups without an icon... Upon editing them, though, and saving, the icon pops up on the list... so I guess it is a very minor detail that runs fine saving an edit, but fails upon triggering a creation

- I've had RDadmin crashes when tweaking with the groups crash ranges... unfortunatelly I was so excited that I failed to start rdadmin from the console to get the stdout/stderr ... my fault ... now I'm trying to reproduce but still not succeeded in seeing the crash.

- I'm experiencing something similar when creating Services: Their details on the list do not show after they've been creted. By editing and saving them, their details show on the list.

Regards!

EDIT got a similar behaviour and a crash while introducing scheduled codes...heres finally a report:

```
ASSERT failure in QList<T>::at: "index out of range", file /usr/include/x86_64-linux-gnu/qt5/QtCore/qlist.h, line 571
Aborted
```

**Comment by ElvishArtisan** [comment]
Fixed in 4e1c92e0. Please test!

**Comment by alexolivan** [comment]
Tested and I can confir Icons do show upon creation.
I feel more consistent CRUDs Create/Update ... Probably is related to other commits in the series.

Not sure If I should Open a new Issue, Keep editing the first post, or opening new ones as I discover UI/UX issues... Let me know how best I could help... Here's another issue I've found:

- Upon creating Scheduler Codes, If you Cancel the creation, the entry seems to have been persited anyways... So, If you try to create the item again, the application complains that the entry already exists, although it doesn't show in the list, which is not intuitive. This can be confirmed by leaving the Scheduler code manager, and entering it again: Then the 'cancelled' entry will show on the list, and can be edited or deleted.

Will keep testing and reporting.
Best regards.

**Comment by ElvishArtisan** [comment]
> Not sure If I should Open a new Issue, Keep editing the first post, or opening new ones as I discover UI/UX issues... Let me know how best I could help... Here's another issue I've found:

Generally, a single Issue per problem is preferred, as that tends to simplfy the long term bookeeping, unless the new bug seems clearly related to the one being addressed. Just use your good engineering judgement.

**Comment by ElvishArtisan** [comment]
> Upon creating Scheduler Codes, If you Cancel the creation, the entry seems to have been persited anyways... So, If you try to create the item again, the application complains that the entry already exists, although it doesn't show in the list, which is not intuitive. This can be confirmed by leaving the Scheduler code manager, and entering it again: Then the 'cancelled' entry will show on the list, and can be edited or deleted.

Confirmed here.

**Comment by ElvishArtisan** [comment]
Fixed in 5ec0d76. Please test!

**Comment by alexolivan** [comment]
Confirm its fixed.
Closing the issue
